code/viprchk.cpp

```diff
@@ -1351,6 +1362,8 @@ bool readConstraint(string &label, int &sense, mpq_class &rhs, shared_ptr<SVecto
          sense = -1;
       else if( senseChar == 'G' )
          sense = 1;
+      else if( label[0] == '%' )
+         return true;
```

Comment:
% right after a label should not be valid, so I would remove this code again.

Reply:
Every line is evaluated in the readConstraint so label contains the first character of the line. If this would be not present every comment would be interpreted as failed constraint.

Reply:
Correct, I was misreading it as sense[0] == "%"